TelepresenceBot/mobile/src/main/res/drawable/ic_arrow_right.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Cheers! I wasn't aware of the `android:rotate` attribute.

I didn't want to create any rotated drawables in xml as I didn't see the benefit over having them as vectors; same amount of files, more effort to make. 🙂